package.json

```diff
@@ -46,10 +46,10 @@
         "LICENSE"
     ],
     "dependencies": {
-        "@autorest/codemodel": "^4.17.0",
+        "@autorest/codemodel": "^4.18.2",
         "@autorest/extension-base": "^3.4.1",
         "@azure-rest/core-client": "1.0.0-alpha.20220112.1",
-        "@autorest/testmodeler": "^1.0.4",
+        "@autorest/testmodeler": "^2.2.4",
```

Comment:
Can we do changes that involve testmodeler in a separate PR?

Reply:
actually if we want to use the security scopes in code model, we need to upgrade the modelerfour version and the older test modeler doesn't compatible with the latest modelerfour. so we need to upgrade them both.
About the testmodeler CG issue, changlong is working on it.

Reply:
Due to https://github.com/Azure/autorest.typescript/issues/1373 we currently don't have the testmodeler dependency so we should be able to have it in a separate PR